File: Java Projects/Java basic to OOP course/first app/src/Class/Student.java
package Class;
public class Student extends University {
    // Properties or Attributes
    private String name;
    private short age;
    // Behaviors
    public Student(String name, int age, String unName){
        this.name = name;
        this.age = (short) age;
        this.unName = unName;
    }
    public void setName(String name){
        this.name = name;
    }
    public String getName(){
        return name;
    }
    public void setAge(short age){
        this.age = age;
    }
    public short getAge(){
        return age;
    }
}

File: CPP Projects/Barmej/Course codes/Student_project/Student.cpp
#include "Student.h"
Student::Student(string name, short grade)
{
    setName(name);
    setGrade(grade);
}
void Student::setName(string n)
{
    name = n;
}
void Student::setGrade(short g)
{
    grade = (g <= 100 && g >=0)? g:grade;
}
string Student::pass()
{
    return (grade >= 60)? "True":"False";
}
void Student::printStatus()
{
    cout << "Student name: " << name << endl
         << "Student grade: " << grade << endl
         << "Student pass: " << pass() << endl;
}